backend/app/Models/Account.php

```diff
@@ -71,4 +72,18 @@ public function getBalance()
 
         return 0;
     }
+
+    public function accountExist(string $accountNumber, string $accountName)
+    {
+        $account = $this->where('account_number', $accountNumber)->first();
+
+        if ($account) {
+            similar_text(Str::lower($account['account_name']), Str::lower($accountName), $percentSimilarity);
```

Comment:
no need to lower

Reply:
@jstephend-sun needs to lower, similar_test is case sensitive. I experienced transferring money using all upper case and was still able to transfer. I think it should also be true to our app.